proposals/001_Core_Contributors_governance.md

```diff
@@ -0,0 +1,66 @@
+# [PROPOSAL-001] Guru Network Core Contributors Incentives and Governance 
+
+## Summary
+
+This proposal outlines the incentive distribution plan for the Guru Network Core Contributors DAO, focusing on community engagement, core contributor rewards. The aim is to ensure transparency, accountability, and sustained growth for the Guru Network.
```

Comment:
Should we name Core Contributors with capitals across ALL document?

Should we introduce who are Core Contributors, and show CC's list in the beginning of the document?